tests/gradients/test_gradient_transform.py

```diff
@@ -135,6 +135,46 @@ def circuit(weights):
         expected = qml.jacobian(circuit)(w)
         assert np.allclose(res, expected, atol=tol, rtol=0)
 
+    def test_multiple_vector_arguments(self, tol):

```

Comment:
It might be worth adding a case for each interface too, to double-check that things work as expected.

Reply:
See my [reply to your second comment](https://github.com/PennyLaneAI/pennylane/pull/1989#discussion_r766067962). What do you think?

Reply:
I think either removing Torch & TF from the logic or keeping them and testing them should work.